srcpkgs/cc65/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'cc65'
+pkgname=cc65
+version=2.18
+revision=1
+short_desc="Freeware C compiler for 6502 based systems"
+maintainer="Nathaniel Barragan <nathanielbarragan@gmail.com>"
+license="Zlib"
+homepage="https://cc65.github.io/"
+distfiles="https://github.com/cc65/cc65/archive/V${version}.tar.gz"
+checksum=d14a22fb87c7bcbecd8a83d5362d5d317b19c6ce2433421f2512f28293a6eaab
+
+do_build() {
+	make PREFIX=/usr
```

Comment:
You should use `make ${makejobs} PREFIX=/usr` to enable parallel builds, unless that would fail with this package.

Reply:
Oops, didn't see the suggested fix to build with gnu-makefile style.